src/gz/bufread.rs

```diff
@@ -304,9 +388,61 @@ pub struct GzDecoder<R> {
     multi: bool,
 }
 
+#[derive(Debug)]
+pub enum GzHeaderParsingState {
+    GzHeaderParsingStart = 0,
+    GzHeaderParsingXlen = 1,
+    GzHeaderParsingExtra = 2,
+    GzHeaderParsingFilename = 3,
+    GzHeaderParsingComment = 4,
+    GzHeaderParsingCrc = 5,
```

Comment:
I think the `= N` and the `GzHeaderParsing` prefix can both be dropped from each of these variants.

Reply:
Ok, will do